src/types.ts

```diff
@@ -12,10 +12,39 @@ export interface WrapperMap {
     readonly timestamp: string;
     readonly version: Version;
     readonly wrappers: {
-        button: WrapperPair[];
-        unbutton: WrapperPair[];
+        [key: string]: WrapperPair[];
     };
     readonly keywords?: string[];
     readonly tags?: Tags;
     readonly logoURI?: string;
+    readonly tokens?: TokenData[];
```

Comment:
WrapperMap shouldn't have tokens property

Reply:
it uses it in `buildList`

Reply:
Oh, buildList should be using the TokenList type rather than the WrapperMap